src/point.rs

```diff
@@ -353,19 +352,19 @@ mod test {
         assert!((actual - expected).abs() <= f32::EPSILON);
 
         let actual = Point { x: 0, y: 0 }.distance((10, 10));
-        let expected = 14.142136;
+        let expected = 14.142_136;
```

Comment:
The number here doesn't actually matter that much. I'm not sure what benefit the extra underscore will provide but I don't mind it there.

Reply:
It's more easy to read, but that's all.